packages/headless/src/provider/apolloClient.ts

```diff
@@ -52,7 +52,7 @@ function createApolloClient(): ApolloClient<NormalizedCacheObject> {
     link: WP_URL
       ? authLink.concat(
           new BatchHttpLink({
-            uri: `${WP_URL}/graphql`,
+            uri: `${WP_URL}/graphql?replace-domain=1`,
```

Comment:
Just a note that we're discussing this and possibly removing the requirement to pass this parameter to the GraphQL endpoint for URL rewrites. We have a setting for enabling/disabling URL rewrites in content, and we think that's sufficient unless customers tell us otherwise.

Reply:
Noting for the record that we dropped the `replace-domain` query string requirement in https://github.com/wpengine/headless-framework/pull/112.

I'll amend this PR to cover the href rewriting only.